scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py

```diff
@@ -295,9 +296,18 @@ def select_data_file(self):
         # these are commneted out as they are a bug
         # see issue 25326
         #self._clear_lines_after_data_file_selected()
-        self._generate_element_widgets()
+        try:
+            self._generate_element_widgets()
+        except ValueError:
+            # todo: add link to elemental analysis documentation after it is merged
```

Comment:
you can merge that code into this branch and then just add put at the top of this PR 

"do not merge until after #xxxx"